install/dc-detect-version.sh

```diff
@@ -40,11 +50,41 @@ fi
 proxy_args="--build-arg http_proxy=${http_proxy:-} --build-arg https_proxy=${https_proxy:-} --build-arg no_proxy=${no_proxy:-}"
 if [[ "$CONTAINER_ENGINE" == "docker" ]]; then
   proxy_args_dc=$proxy_args
+  dcr="$dc run --pull=never --rm"
 elif [[ "$CONTAINER_ENGINE" == "podman" ]]; then
   proxy_args_dc="--podman-build-args http_proxy=${http_proxy:-},https_proxy=${https_proxy:-},no_proxy=${no_proxy:-}"
+  dcr="$dc run --rm"
 fi
-dcr="$dc run --pull=never --rm"
 dcb="$dc build $proxy_args"
 dbuild="$CONTAINER_ENGINE build $proxy_args"
 
+# Utility function to handle --wait with docker and podman
+function start_service_and_wait_ready() {
```

Comment:
Well well...

@aldy505 - with this, maybe we can lower the Docker Compose version requirement again as some people were complaining about that?

Reply:
Yes we might. Do you have any thoughts regarding this? @aminvakil

Reply:
Yes I agree, let's leave it for another PR then.